cmd/function/access_test.go

```diff
@@ -326,6 +327,7 @@ func TestFunctionDistributionCommand_PermissionsFile(t *testing.T) {
 
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {
+			ctx := slackcontext.MockContext(t.Context())
```

Comment:
thought: It's generally best to declare the `ctx` at the top of the function, so that the same context is used throughout.